scss/bootstrap/overrides/style/_forms.scss

```diff
@@ -198,3 +191,10 @@ label {
     top: 0;
   }
 }
+
+@media (min-width: 768px) {
+  .form-inline .radio input[type="radio"],
```

Comment:
to better organize the code, put first the `.form-inline` styling (and nesting the media query) and then you put the styling for `.form-inline-tabs`. So you have the default version first and the the variation
```scss
.form-inline {

  @media () {
    // ...
  }
}

.form-inline-tabs {

}